react/index.js

```diff
@@ -61,15 +51,6 @@ export { default as IntentHeader } from './IntentHeader'
 export { default as Panel } from './Panel'
 export { default as SelectBox, SelectBoxWithFixedOptions } from './SelectBox'
 export { default as Chip } from './Chip'
-export {
-  default as Text,
-  MainTitle,
-  Title,
-  SubTitle,
-  Bold,
-  Caption,
-  Uppercase
-} from './Text'
```

Comment:
this commit is also a BC since we don't expose anymore Text or Modal from the root of UI, right?

Reply:
Yep, I've added the BC in the commit.